flow/layers/layer_tree.cc

```diff
@@ -64,6 +64,7 @@ bool LayerTree::Preroll(CompositorContext::ScopedFrame& frame,
       .checkerboard_offscreen_layers = checkerboard_offscreen_layers_,
       .frame_device_pixel_ratio      = device_pixel_ratio_,
       .raster_cached_entries         = &raster_cache_items_,
+      .display_list_enabled          = frame.display_list_builder() != nullptr,
```

Comment:
So if I'm understanding this right, this always ends up being true for Impeller because Impeller's `Surface`s always produce `SurfaceFrame`s that have a display list builder, right?

Reply:
Yes, it is true iff Impeller is being used.

We _might_ eventually switch to all DLs all the time even for Skia, but that remains to be seen.